util/chunk/column.go

```diff
@@ -71,6 +71,11 @@ func NewColumn(ft *types.FieldType, cap int) *Column {
 	return newColumn(getFixedLen(ft), cap)
 }
 
+// NewDefaultColumn creates a new default column with the 8 length and InitialCapacity.
+func NewDefaultColumn() *Column {
```

Comment:
Is necessary to add this?

Reply:
To reduce 2 function calls

Reply:
How about this?
```
func newLocalColumnPool() *localColumnPool {
	emptyColumn := chunk.NewColumn(types.NewFieldType(mysql.TypeLonglong), chunk.InitialCapacity)
	return &localColumnPool{
		sync.Pool{
			New: func() interface{} {
				return emptyColumn.CopyConstruct(nil)
			},
		},
	}
}
```
`CopyConstruct` can create a new Column.